ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1576,6 +1576,11 @@ vxlan_interface:
       mlag_source_interface: < source_interface_name >
       udp_port: < udp_port >
       virtual_router_encapsulation_mac_address: < mlag-system-id | ethernet_address (H.H.H) >
+      bfd_vtep_evpn:
+        interval: < integer >
+        min_rx: < integer >
```

Comment:
Here not sure if this makes sense to change it to the allowed range - my vEOS CLI is `< 50 - 60000 >` but not sure if they are the same for every platforms. I am thinking that when `schema is coming` we can then detect error range and having already the values in the README could help

PS: this can be ignored